examples/apps/simple_wallet/README.md

```diff
@@ -0,0 +1,81 @@
+# Simple Wallet Demo Application
+
+This application demostrates basic wallet transactions like deposit, withdraw and transfer
+by accessing encrypted wallet ledger file using inside-out API and applying crypto operations
+on the ledger data.
+
+## Using the Generic Command Line Client to execute Simple Wallet operations
+
+Generic command line client, `generic_client.py` sends an input request
+with transaction type, account identifier(s), amount and encryption key to the worker
+which access ledger which contains all wallet details in encrypted format using inside-out API.
+The encrypted ledger data is then decrypted by the workload by making use of crypto functions.
+Once the balance is updated, the result is encrypted and written to ledger using file i/o.
+
+Below are the syntax for supported wallet transactions.  
+```
+1. deposit <account_identifier> <amount> <encryption_key_in_hex>
+2. withdraw <account_identifier> <amount> <encryption_key_in_hex>
+3. transfer <debitor_identifier> <creditor_identifier> <amount> <encryption_key_in_hex>
+```
+
+In the above transactions, same hex encoded symmentric key of length 64 needs to be supplied 
+which will be used for for encryption/decryption of ledger data.
+
+
+For command line options, type `./generic_client.py -h` from the
+`$TCF_HOME/examples/apps/generic_client/client` directory.
+
+To use:
+
+1.  If needed, update the Ethereum account and direct registry contract
+    information in `docker/Dockerfile.tcf-dev` and
+    `examples/common/python/connectors/tcf_connector.toml`
+2.  Follow instructions in the "Docker-based Build and Execution" section of
+    the [build document](../../../BUILD.md#dockerbuild) through step 4
+3.  Terminal 1 is running the TCF Enclave Manager and Listener with
+    `docker-compose` and Terminal 2 is running the Docker container.
+4.  In Terminal 2 run `cd $TCF_HOME/examples/apps/generic_client` 
+
+5. Execute wallet transactions.
+
+Deposit transaction:  
+
+In Terminal 2, Create wallets for Alice and bob  
+
+```
+./generic_client.py --workload_id "simple-wallet" --in_data "deposit alice 100 
+aabbccddaabbccddaabbccddaabbccddaabbccddaabbccddaabbccddaabbccdd"  
```

Comment:
In this example, please explain where I get or generate the encryption key used for this example.